coriolis/osmorphing/base.py

```diff
@@ -342,3 +343,28 @@ def _set_selinux_autorelabel(self):
                 "touch /.autorelabel")
         except Exception as err:
             LOG.warning("Failed to set autorelabel: %r" % err)
+
+    def _retain_user_credentials(self):
```

Comment:
This method name is pretty uninformative.
Please rename it to `_configure_cloud_init_user_retention` or something clearly indicating it's only cloud-init related.